experimental/ConsoleApp6/ConsoleApp6/Examples.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+
+// Example library that creates some simple metrics with a few default labels
+namespace SquidLibrary
+{
+    class SquidAnalyzer
+    {
+        static readonly MetricSource metrics = new MetricSource("SquidLibrary",
```

Comment:
Need to discuss if we really want to have a "Provider" as the starting point.

I think it makes more sense to just "new Counter(...)" immediately.  And employ late binding to attach to a "provider".